src/pages/MerchantGiftCards/MerchantGiftCards.tsx

```diff
@@ -0,0 +1,145 @@
+import React, { useState, useEffect } from 'react';
+import { useHistory } from 'react-router-dom';
+import {
+  getMerchantGiftCards,
+  getSeller,
+} from '../../utilities/api/interactionManager';
+import Loader from '../../components/Loader/Loader';
+import styles from './styles.module.scss';
+import ErrorPage from '../../components/404Page';
+const FilterableTable = require('react-filterable-table');
+
+const MerchantGiftCards = () => {
+  const params = useHistory();
+  let urlParams = params.location.pathname.match(/\/[^/]+/g) as string[];
+  urlParams = urlParams.map((param) => param.replace('/', ''));
+  const metadata = {
+    sellerId: urlParams[0],
+    secretId: urlParams[2],
+  };
+
+  const [error, setError] = useState(false);
+  const [loading, setLoading] = useState(true);
+  const [giftCards, setGiftCards] = useState<any | null>();
+  const [seller, setSeller] = useState<any | null>();
+
+  const renderAmount = (props) => {
+    return Intl.NumberFormat('en-US', {
+      style: 'currency',
+      currency: 'USD',
+      maximumSignificantDigits: 4,
+    }).format(props.value / 100);
+  };
+
+  const renderDate = (props) => {
+    return new Date(props.value).toISOString().substring(0, 10);
+  };
+
+  const fields = [
```

Comment:
seems like a static var that we could maybe move above/below the react component, it would be easier to read inner workings of the react component with the less clutter of these static values.

Reply:
Good call! I restructured the file to be a bit more readable now I think 👌